packages/admin-ui/src/params.ts

```diff
@@ -111,6 +111,11 @@ export const autoUpdateIds = {
   SYSTEM_PACKAGES: "system-packages"
 };
 
+// IPFS
+export const IPFS_DAPPNODE_GATEWAY = "http://ipfs.dappnode.io:8081";
```

Comment:
The UI should not be aware of this. Why is this param necessary?

Reply:
I see helpful from a user point of view to have there always the dappnode "trusted" gateway in case the changed it and want to use it again